ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,59 @@
+<!--
+Compensation requests have a specific format that enables them to be parsed programmatically for more efficient budgeting. Please make sure you stick to the template. A linter will evaluate your request after you remove [WIP] from the issue title and notify you of any issues.
+
+See the following link for details:
+https://bisq.wiki/Making_a_compensation_request
+
+Feel free to browse other requests if you're not sure about something, or reach out in #compensation on Keybase (https://keybase.io/team/bisq).
+-->
+
+
 ## Summary
 
-> _Specify the total amount of BSQ you are requesting:_
+> _Specify the total amount of BSQ you are requesting, along with the USD total and BSQ/USD rate:_
 
- - BSQ requested:
- 
- <!--
- Don't forget to make your compensation request in the actual Bisq DAO! See the following link for details:
- https://docs.bisq.network/compensation.html
- -->
+- **BSQ requested**: **`[BSQ amount]`** <!-- [BSQ amount] should be: USD amount / BSQ rate -->
+- USD requested: [USD amount] <!-- [USD amount] should be: total of all contributions listed in "contributions delivered" table below -->
+- BSQ rate: [rate] USD per BSQ <!-- [rate] should be the rate used to determine the BSQ amount above, as specified in the cycle's rate announcement (https://bisq.wiki/Compensation_Maintainer#Announce_BSQ-USD_rate) -->
 
 
 ## Contributions delivered
 
-> _Provide links to contributions you have [delivered](https://github.com/bisq-network/proposals/issues/19), the amount of BSQ you are requesting for each, and any comments that will help stakeholders understand its value._
+> _Add contributions you have [delivered](https://github.com/bisq-network/proposals/issues/19) here as new rows in the table below. See comments below the table for guidance._
```

Comment:
"See comments below the table for guidance" shows up in the rendered issue, but the comments being referred to do not because they are HTML comments. This leads to a situation like we see in bisq-network/compensation#583, where @jmacxx left the "See comments below" text in place, but then there are no comments to be seen. This is confusing for the uninitiated reader. I immediately scrolled down looking for the comments and was frustrated not to find them. I'm sure others will have the same experience.

Quick solution would be to just put the "See comments..." text in an HTML comment so that the submitter sees it only when creating / editing the issue.

Reply:
Addressed with 005d1833f8bf0f816d22bbdbef9adc1768b429d4. Thanks for pointing it out.